ebook_converter/constants_old.py

```diff
@@ -32,7 +32,7 @@ def debug():
 # plugins {{{
 
 
-class Plugins(collections.Mapping):
+class Plugins(collections.abc.Mapping):
```

Comment:
Aww, I missed this for some reason. Obviously there was warning regarding `Mapping` and `MutableSet`